Print_A_Bot/djwifi/management/commands/system_config.py

```diff
@@ -0,0 +1,273 @@
+import re
```

Comment:
this is a custom django management command
https://docs.djangoproject.com/en/1.11/howto/custom-management-commands/